packages/contracts/contracts/L1/rollup/CanonicalTransactionChain.sol

```diff
@@ -539,45 +498,6 @@ contract CanonicalTransactionChain is ICanonicalTransactionChain, Lib_AddressRes
         );
     }
 
-    /**
```

Comment:
You can also remove `_getSequencerLeafHash` and `_getQueueLeafHash`

Reply:
Done!